ippon/event/views.py

```diff
@@ -28,7 +28,7 @@ def create(self, request: Request, *args, **kwargs) -> Response:
         url_name='my_tournaments')
     def my_tournaments(self, request: Request):
```

Comment:
I don't like that name. Maybe some verb? This is supposed to be user's tournaments, right? If yes, maybe we could try using query parameters to filter events or sth...
And this is not returning even tournaments.